src/rust/engine/process_execution/src/lib.rs

```diff
@@ -225,13 +229,19 @@ pub struct Process {
   #[derivative(PartialEq = "ignore", Hash = "ignore")]
   pub description: String,
 
-  // This will be materialized for local Process only.
-  // Eventually we want to remove this.
-  // Context: https://github.com/pantsbuild/pants/issues/8314
-  // Think twice before using it.
-  #[derivative(PartialEq = "ignore", Hash = "ignore")]
-  pub unsafe_local_only_files_because_we_favor_speed_over_correctness_for_this_rule:
-    hashing::Digest,
+  ///
```

Comment:
This is not your problem, but this change highlights the paucity of docs on the python side Process where its arguably needed more.

Reply:
Yea, I think that all of the docs should likely move to the python side in a followup change.